library/src-bootstrapped/scala/quoted/Unliftable.scala

```diff
@@ -8,368 +8,365 @@ trait Unliftable[T] {
    *  Returns `None` if the expression does not contain a value or contains side effects.
    *  Otherwise returns the `Some` of the value.
    */
-  def apply(x: Expr[T])(using qctx: QuoteContext): Option[T]
+  def fromExpr(x: Expr[T]): QuoteContext ?=> Option[T]
```

Comment:
What about the name `from`? `unlift.from(expr)` reads better than `unlift.fromExpr(expr)`.

```suggestion
  def from(x: Expr[T]): QuoteContext ?=> Option[T]
```

Reply:
That would not align with the `Liftable` and we almost never need to write it explicitly.

Reply:
> That would not align with the `Liftable` and we almost never need to write it explicitly.

It makes sense to be consistent.